typography.html

```diff
@@ -18,10 +18,6 @@
 
     /* Shared Styles */
 
-    /*
```

Comment:
If you are removing this comment, then you should probably remove the commented out `@applies` for these styles in the mixins in this file.